tests/functional/build_command/test_build.py

```diff
@@ -73,14 +75,14 @@ def models(self):
             "model_2.sql": models__model_2_sql,
             "model_3.sql": models__model_3_sql,
             "model_99.sql": models__model_99_sql,
-            "test.yml": models__test_yml,
+            "test.yml": models__test_yml + unit_tests__yml,
         }
 
     def test_failing_test_skips_downstream(self, project):
         results = run_dbt(["build"], expect_pass=False)
-        assert len(results) == 13
+        assert len(results) == 14
         actual = [str(r.status) for r in results]
-        expected = ["error"] * 1 + ["skipped"] * 5 + ["pass"] * 2 + ["success"] * 5
+        expected = ["error"] * 1 + ["skipped"] * 6 + ["pass"] * 2 + ["success"] * 5
```

Comment:
this will need to be updated once we fix the order of unit tests -> model -> data tests, as expected 👍